.happy/terraform/envs/dev/main.tf

```diff
@@ -22,6 +24,14 @@ module stack {
   frontend_memory              = 4096
   backend_instance_count       = 4
   backend_workers              = 1
+  backend_de_instance_count     = 4
```

Comment:
These are a bit verbose now - maybe grouping them into an object would make it cleaner.